src/Plugin/PhpCsFixer.php

```diff
@@ -58,8 +63,19 @@ public function __construct(Builder $builder, Build $build, array $options = [])
             $this->args   .= ' --config=' . $builder->interpolate($options['config']);
         }
 
-        if (isset($options['directory']) && $options['directory']) {
-            $this->directory = $builder->interpolate($options['directory']);
+        $this->directory = isset($options['directory'])
+            ? $options['directory']
```

Comment:
I guess code should be: `? $builder->interpolate($options['directory'])`.

Reply:
I do the same code than in Phan plugin...

Reply:
Values like `%BUILD_PATH%/directory` with [special variables](https://github.com/php-censor/php-censor/blob/master/docs/en/interpolation.md) will not work without interpolation. Phan plugin should also have interpolation but something went wrong :)